capstone/src/main/resources/static/eventDetails.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Event Details</title>
+  <script src="event.js"></script>
+</head>
+<body onload="getEventDetails();">
```

Comment:
Is this file used anywhere? It seems like you decided to go with a modal popup instead of a separate page.